components/player-card/player-recent-matches.tsx

```diff
@@ -49,6 +59,18 @@ const PlayerRecentMatches = ({
     return matchRecord.matchhistoryreportresults[0];
   };
 
+  const renderMap = (name: any) => {
+    return (
+      <Card style={{ background: "none" }}>
+        <Card.Section>
+          <Center>
+            <Image src={maps[name]?.url} width={160} />
```

Comment:
Can you please change this to Image from nextJS, the way we have it here:
https://github.com/cohstats/coh3-stats/blob/master/components/country-flag.tsx#L5
and add parameter loading="lazy" << this is important so it loads super fast.

Reply:
Sure, done.